### PR TITLE
Form hendler

### PR DESCRIPTION
**Pull request by GoodFellow23**
Issue: https://trello.com/c/x1IQdL7d

### CODE REVIEW

File: poker/src/components/Checkbox/Checkbox.tsx [hunk 1]
```diff
@@ -3,7 +3,7 @@ import styles from './Checkbox.module.scss';
 
 interface Props {
   name: string;
-  value?: string;
+  value?: boolean;
```
Comment (AnAtoliy-AA) [inline poker/src/components/Checkbox/Checkbox.tsx]: Bad name

Comment (GoodFellow23) [reply poker/src/components/Checkbox/Checkbox.tsx]: fixed
a82e377

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Local Netlify folder
+.netlify
+.idea
```
Comment (AnAtoliy-AA) [inline .gitignore]: empty line

File: poker/src/components/ConnectToLobby/ConnectToLobby.tsx [hunk 3]
```diff
@@ -1,7 +1,15 @@
...
+import { useDispatch, useSelector } from 'react-redux';
+import { Socket } from 'socket.io-client/build/socket';
+import { DefaultEventsMap } from 'socket.io-client/build/typed-events';
+import { useHistory } from 'react-router-dom';
 import GeneralButton from '../GeneralButton/GeneralButton';
 import InputComponent from '../InputComponent/InputComponent';
 import Checkbox from '../Checkbox/Checkbox';
+import { ROLES, User } from '../../types/common';
+import { updateUser } from '../../redux/slices/userSlice';
+
```
Comment (AnAtoliy-AA) [inline poker/src/components/ConnectToLobby/ConnectToLobby.tsx]: remove empty line

File: poker/src/components/ConnectToLobby/ConnectToLobby.tsx [hunk 4]
```diff
@@ -1,7 +1,15 @@
...
+import { SocketContext } from '../../socketContext';
```
Comment (AnAtoliy-AA) [inline poker/src/components/ConnectToLobby/ConnectToLobby.tsx]: remove empty line

**Review by AnAtoliy-AA — APPROVED** [review]

File: poker/src/components/ConnectToLobby/ConnectToLobby.tsx (hunk 4 above)
Comment (GoodFellow23) [reply poker/src/components/ConnectToLobby/ConnectToLobby.tsx]: fixed
f67a593

File: poker/src/components/ConnectToLobby/ConnectToLobby.tsx (hunk 3 above)
Comment (GoodFellow23) [reply poker/src/components/ConnectToLobby/ConnectToLobby.tsx]: fixed
f67a593

File: .gitignore (hunk 2 above)
Comment (GoodFellow23) [reply .gitignore]: fixed
f67a593

**Review by Vlad-coder-678 — APPROVED** [review]

File: poker/src/types/common.ts [hunk 5]
```diff
@@ -28,3 +28,11 @@ export interface SocketError {
   error: string;
   eventName: string;
 }
+
+export interface User {
+  firstName: string;
+  lastName: string;
+  jobPosition: string;
```
Comment (Evgenia-cyber) [inline poker/src/types/common.ts:35]: lastName and jobRosition not required

File: poker/src/types/common.ts [hunk 6]
```diff
@@ -28,3 +28,11 @@ export interface SocketError {
   error: string;
   eventName: string;
 }
+
+export interface User {
```
Comment (Evgenia-cyber) [inline poker/src/types/common.ts:32]: interface User must have userId

**Review by Evgenia-cyber — APPROVED** [review]

**Pull request merged**